### ISSUE TITLE
Redstone Furnace (Thermal Expansion) produces infected Charcoal from all log types

### ISSUE THREAD

**Issue by CMastar** [SteveKunG/MorePlanets#87]
Abruptly happened after a server restart, redstone furnace switched to producing infected charcoal instead of charcoal from all log types. JEI still indicates that infected charcoal should only come from infected log types.

May be a TE issue rather than a More Planets one.

More Planets 1.12.2-2.0.24-GC251
Thermal Expansion 1.12.2-5.5.7.1

Running on Magma with FML version 14.23.5.2847

**Comment by SteveKunG** [comment]
Legacy version of More Planets 1.12.2 (Below 2.0.x) is no longer supported due to bad codebase and maintenance cost. So it's not worth it to support anymore.
I'd suggest to use newer version (2.1.x+) instead. Thank you.

**Comment by onlikerop** [comment]
Hi there!
Got the same problem on singleplayer Forge 1.12.2 with:
Thermal Expansion 1.12.2-5.5.7.1
More Planets 1.12.2-2.3.3-GC4.0.6.

I was able to sort out the problem and want my solution to remain in history.
1. I cleared all redstone furnaces from the wood and infected coal and stopped their auto-input. // Without this step the steps below did not work for me
2. I removed More Planets and started the game (without entering the world ofc).
3. I did install More Plantes 2.3.3 back again
4. I started the game and entered the world.
5. It worked
